### PR TITLE
[16.0][FIX] stock_account_move_reset_to_draft: Update the README

### PR DESCRIPTION
**Pull request by kanda999**
Corrected inaccurate information in the README.

@qrtl QT4200

### CODE REVIEW

File: stock_account_move_reset_to_draft/readme/USAGE.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
-#. Create a product category with Costing Method: Average Cost (AVCO)
+#. Create a product category with Costing Method: Average Cost (AVCO) or First In First Out (FIFO).
 #. Create a product linked to the category created before.
 #. Create a purchase order and adds a product line with quantity 1 and price 10.
 #. Confirms the purchase order and validates the incoming picking.
 #. Creates an invoice from the purchase order.
 #. Changes the invoice line price to 12 and confirm the invoice.
-#. It is possible to reset the invoice to draft (the SVL of the difference will be removed).
+#. It is possible to reset the invoice to draft (the record to offset the SVL difference will be created).
```
Comment (yostashiro) [inline stock_account_move_reset_to_draft/readme/USAGE.rst]:
```suggestion
#. It is possible to reset the invoice to draft (a new SVL record to offset the existing SVL difference will be created).
```

**Review by yostashiro — COMMENTED** [review]
@kanda999 Please adjust DESCRIPTION.rst as well.

File: stock_account_move_reset_to_draft/readme/USAGE.rst (hunk 1 above)
Comment (yostashiro) [reply stock_account_move_reset_to_draft/readme/USAGE.rst]: We may also add this explanation: https://odoo-community.org/groups/contributors-15/contributors-1965466

File: stock_account_move_reset_to_draft/README.rst [hunk 2]
```diff
@@ -39,21 +42,32 @@ have been generated (stock_account not allowed).
...
+#. Create a product category with Costing Method: Average Cost (AVCO) or First In First Out (FIFO).
 #. Create a product linked to the category created before.
 #. Create a purchase order and adds a product line with quantity 1 and price 10.
 #. Confirms the purchase order and validates the incoming picking.
 #. Creates an invoice from the purchase order.
 #. Changes the invoice line price to 12 and confirm the invoice.
-#. It is possible to reset the invoice to draft (the SVL of the difference will be removed).
+#. It is possible to reset the invoice to draft (a new SVL record to offset the existing SVL difference will be created).
+
+When attempting to reset an invoice to draft for a product that has been partially or fully consumed in manufacturing,
```
Comment (yostashiro) [inline stock_account_move_reset_to_draft/README.rst]:
```suggestion
When attempting to reset an invoice to draft for a product that has been partially or fully consumed, 
```

File: stock_account_move_reset_to_draft/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -1,2 +1,5 @@
-This module allows to restore a vendor bill to draft if SVLs linked to any invoice line
-have been generated (stock_account not allowed).
+This module allows a vendor bill to be reverted to draft status 
+when an SVL (Stock Valuation Layer) is created due to a price difference between the receipt and the vendor bill. 
+(In the standard Odoo stock_account module, this is not allowed.)
+Additionally, when necessary, a new SVL record is generated to adjust the valuation difference accordingly.
```
Comment (yostashiro) [inline stock_account_move_reset_to_draft/readme/DESCRIPTION.rst]:
```suggestion
This module allows a vendor bill to be reverted to draft status even
when an SVL (Stock Valuation Layer) record is created due to a price difference between the receipt and the vendor bill,
when inventory from the receipt has yet to be consumed.
When a vendor bill is reset to draft, as applicable, a new SVL record is generated to offset the valuation difference that was generated upon the bill's confirmation.
```

**Review by yostashiro — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]

**Pull request merged**